it/v3/endpoints/IgnoreEmploymentExpensesControllerISpec.scala

```diff
@@ -51,9 +50,7 @@ class IgnoreEmploymentExpensesControllerISpec extends IntegrationBaseSpec {
         }
 
         val response: WSResponse = await(request().post(requestBody))
-        response.status shouldBe OK
-        response.json shouldBe responseBody
```

Comment:
this can be replaced to verify that there is no body
response.body.isEmpty shouldBe true